core/domain/change_domain.py

```diff
@@ -94,12 +103,18 @@ class BaseChange(python_utils.OBJECT):
     # (a list of required attribute names of a command),
     # optional_attribute_name (the list of optional attribute names of a
     # command), user_id_attribute_names (the list of attribute names that
-    # contain user ID). There can be a optional key allowed_values which is a
+    # contain user ID). There are also optional keys like allowed_values
+    # and deprecated_values. allowed_values is a
     # dict with key as attribute name and value as allowed values
+    # for the attribute. deprecated_values is a
+    # dict with key as attribute name and value as deprecated values
     # for the attribute.
     # This list can be overriden by subclasses, if needed.
     ALLOWED_COMMANDS = []
 
+    # The list of deprecated commands of a change domain object.
```

Comment:
Add a better explanation of what are deprecated commands and what should we do with them after they are deprecated.

Reply:
Done.